src/stp/PlayDecider.cpp

```diff
@@ -5,12 +5,19 @@
 #include "stp/PlayDecider.hpp"
 
 namespace rtt::ai::stp {
-Play *PlayDecider::decideBestPlay(world::World *pWorld, std::vector<Play *> plays) noexcept {
-    if (lockedPlay) {
-        return lockedPlay;
+
+    bool PlayDecider::change = false;
+
+    Play *PlayDecider::decideBestPlay(world::World *pWorld, std::vector<Play *> plays) noexcept {
+        if (lockedPlay) {
+            return lockedPlay;
+        }
+        return *std::max_element(plays.begin(), plays.end(), [&](auto &largest, auto &play) { return largest->score(pWorld) < play->score(pWorld); });
     }
-    return *std::max_element(plays.begin(), plays.end(), [&](auto &largest, auto &play) { return largest->score(pWorld) < play->score(pWorld); });
-}
 
-void PlayDecider::lockPlay(Play *play) { lockedPlay = play; }
+    // This is only used by the interface to force new plays
+    void PlayDecider::lockPlay(Play *play) {
+        PlayDecider::change = true;
+        lockedPlay = play;
```

Comment:
Ah this is where it changes play, consider removing the `bool change`, instead just use the play*.